src/Quantum/misc_known_codes.jl

```diff
@@ -8,6 +8,46 @@
       # Subsystem codes
 #############################
 
+"""
+[[9, 1, 4, 3]] gauged Shor code from [poulin2005stabilizer](@cite)
+
+```jldoctest
+julia> using CodingTheory, Oscar;
+
+julia> code = GaugedShorCode();
+
+julia> length(code), dimension(code), code.r
+(9, 1, 4)
+
+julia> code.stabs_stand
```

Comment:
No public facing code should ever use the dot operator. Use the getter functions for all of these.

Reply:
In fact, using the dot operator explicitly will break a significant amount of the code.